tests/unit/components/bs-form-test.js

```diff
@@ -0,0 +1,91 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+import { next, run } from '@ember/runloop';
+import RSVP from 'rsvp';
+
+const nextRunloop = function() {
+  return new RSVP.Promise((resolve) => {
+    next(() => resolve());
+  })
+};
+
+module('Unit | Component | bs-form', function(hooks) {
```

Comment:
Hope it's okay to use a unit test for `isSubmitting` property. Had the feeling that it's a little bit easier to write and read. Could refactor to integration test if you would prefer that one.

Reply:
Hm, tests look good, but I indeed think an integration test would be better. See my other comment about making `isSubmitting` private, which would require to test for the yielded value, and not the property itself.